.gitignore

```diff
@@ -2,10 +2,10 @@
 /*/*/
 
 # Except some Windows folders
-!/windows/icu-easyrpg/*
+!/windows/icu-easyrpg/
```

Comment:
With the "*" I get a warning that the folder is ignored. Without the "*" it works

Reply:
I remember i had a similar problem, that is why i changed it here too:
https://github.com/EasyRPG/buildscripts/pull/55/files#diff-a084b794bc0759e7a6b77810e01874f2